lighthouse-core/audits/dobetterweb/notification-on-start.js

```diff
@@ -49,7 +49,7 @@ class NotificationOnStart extends ViolationAudit {
     const results = ViolationAudit.getViolationResults(artifacts, /notification permission/);
 
     const headings = [
-      {key: 'url', itemType: 'url', text: 'URL'},
+      {key: 'url', itemType: 'url', text: 'URL', styleAs: 'url'},
```

Comment:
Another approach I was exploring is to allow headers to define their own styling: `style: "width:50px"`, then use that in the renderer. It does pollute the audits a bit with rendering information, but it may be more flexible in the long-term. Also cool with sticking with classes for now.